benchmarks/apache-spark/src/main/scala/org/renaissance/apache/spark/SparkUtil.scala

```diff
@@ -59,4 +61,10 @@ trait SparkUtil {
       System.setProperty("hadoop.home.dir", tempDirPath.toAbsolutePath.toString)
     }
   }
+
+  def checkCaching[T](rdd: RDD[T]): Unit = {
+    if (!rdd.getStorageLevel.useMemory) {
```

Comment:
`rdd.getStorageLevel.useMemory` is true when `cache()` has been called on the RDD.